xcodeproj/internal/top_level_targets.bzl

```diff
@@ -225,6 +226,9 @@ def process_top_level_target(
     additional_files.extend(list(modulemaps.files))
 
     app_icon_info = app_icons.get_info(ctx, automatic_target_info)
+    alternate_icons_info = alternate_icons.get_info(ctx, automatic_target_info)
```

Comment:
Instead of this function, you can do the one part of the function where you collect the files. Then you can delete that bzl file.